zombienet/local-testnet.toml

```diff
@@ -27,7 +27,13 @@ id = 1000
 
 # run charlie as parachain collator
 [[parachains.collators]]
-args = ["--detailed-log-output", "--pool-type=fork-aware", "-lparachain=debug,xcm=trace,runtime=trace,txpool=debug,basic-authorship=debug"]
+args = [
+  "--detailed-log-output",
+  "--p2p-key=@zombienet/private.pem",
+  "--p2p-listen-address=/ip4/127.0.0.1/tcp/62649",
+  "--pool-type=fork-aware",
+  "-lparachain=debug,xcm=trace,runtime=trace,txpool=debug,basic-authorship=debug",
+]
```

Comment:
It does also need changes to `maat/src/lib.rs` , we pass those args manually as well.

Reply:
+ `kube-testnet` as well.